swift/apple/FirezoneKit/Sources/FirezoneKit/Managers/VPNConfigurationManager.swift

```diff
@@ -301,38 +295,21 @@ public class VPNConfigurationManager {
       options.merge(["id": id as NSObject]) { _, new in new }
     }
 
-    try session.startTunnel(options: options)
+    try session().startTunnel(options: options)
   }
 
   func signOut() throws {
-    guard let session = session()
-    else { throw VPNConfigurationManagerError.managerNotInitialized }
-
-    guard [.connected, .connecting, .reasserting].contains(session.status)
-    else { throw VPNConfigurationManagerError.invalidStatus(session.status) }
-
-    session.stopTunnel()
-    try session.sendProviderMessage(encoder.encode(TunnelMessage.signOut))
+    try session([.connected, .connecting, .reasserting]).stopTunnel()
+    try session().sendProviderMessage(encoder.encode(TunnelMessage.signOut))
```

Comment:
Bit odd to call the function twice, why not assign a variable?

Reply:
The first call enforces the VPN to be in one of those three states, the second simply calls the IPC function to delete the saved token.

Reply:
This could be refactored though to pass `clearToken: Bool` as an argument to the `stop` call, but I'll save that for a future PR.